administration/package.json

```diff
@@ -17,6 +17,7 @@
     "@zxing/library": "^0.19.2",
     "apollo-upload-client": "^17.0.0",
     "date-fns": "^2.29.3",
+    "date-fns-tz": "^2.0.0",
```

Comment:
I think we don't need this dependency and we can just use [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat). We don't need to support IE11 :D